src/frontend/packages/lib_video/src/components/common/VideoWidgetProvider/widgets/DeleteVideo/index.tsx

```diff
@@ -55,6 +55,11 @@ const messages = defineMessages({
     description: 'ext of the delete error toast..',
     id: 'components.videoDeleteError.text',
   },
+  deleteModalTitle: {
+    defaultMessage: 'Delete resource',
+    description: 'Title of the delete modal.',
+    id: 'components.DeleteVideoButton.modalTitle',
```

Comment:
To be sure you don't repeat the id, usually it is more something like
`id: '[folder].[component].[nameVar]'`.
```suggestion
    id: 'components.DeleteVideo.deleteModalTitle',
```